File: 251004/Snopko/Publisher/src/main/java/com/example/restservice/dto/request/MarkRequestTo.java
package com.example.restservice.dto.request;

import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.Size;
import lombok.Data;

import java.time.LocalDateTime;

@Data
public class MarkRequestTo {
    @NotBlank(message = "Name cannot be blank")
    @Size(min = 2, max = 32, message = "Name must be between 2 and 32 characters")
    private String name;

    private Long id;

}

File: 251003/Kukhatskavolets/microblog/src/main/kotlin/by/kukhatskavolets/dto/requests/MarkRequestTo.kt
package by.kukhatskavolets.dto.requests

import jakarta.validation.constraints.NotBlank
import jakarta.validation.constraints.Size

data class MarkRequestTo(
    val id: Long = 0,

    @field:NotBlank(message = "Name may not be blank")
    @field:Size(min = 2, max = 32, message = "Name should be between 2 and 32 symbols")
    val name: String = ""
)